etc/workflows/fast.yaml

```diff
@@ -44,6 +44,16 @@ operations:
       - target-tags: "engage-download,engage-streaming,rss,atom"
       - encoding-profile: "fast.http"
 
+  - id: tag
+    fail-on-error: true
+    exception-handler-workflow: "partial-error"
+    description: "Tag captions for publication"
+    configurations:
+      - source-flavor: "captions/source"
+      - target-flavor: "captions/preview"
+      - target-tags: "+engage-download"
+      - copy: true
```

Comment:
Should you add?
`generator:unknown`
`generator-type:manual`
`lang:unknown`

Reply:
It is not necessary per se, tags are not variables that need to be initalized or anything like that. 

I don't think the workflows generally should make general assumptions about subtitle tags (was the caption written manually or automatically, the workflow does not know). Although since this is a workflow used primarily for testing, maybe the tags are nice for "demonstrating" how tags for captions are used in Opencast?

Bottom line, for the fast workflow in particular I don't really care. If you think it's a good idea, let's do it.

Reply:
Fair enough. I'm only thinking in your changes in Paella, that can detect those tags for the subtitle selection.